features/signal/prevent_close/README.md

```diff
@@ -0,0 +1,18 @@
+# Pending signals prevents workflows from closing
+
+In Temporal, any pending signal for a given workflow execution will prevent that
+workflow execution from completing.
+
+This feature tests that a signal queued for processing at the time that a
+workflow would otherwise exit in fact keeps the workflow running until the
+signal has been delivered and processed.
+
+# Detailed spec
+
+In most cases, having run the workflow function to completion, the SDK will send
+a `RespondWorkflowTaskCompletedRequest` to the server containing a command of
+type `COMMAND_TYPE_COMPLETE_WORKFLOW_EXECUTION` to indicate that the workflow
```

Comment:
Or any other command which would complete workflow execution - failure and continue as new both apply too